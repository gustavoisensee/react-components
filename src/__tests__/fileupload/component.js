import React from 'react';
import renderer from 'react-test-renderer';
import FileUpload from './../../modules/fileupload/component';

describe('FileUpload component test', () => {
  test('FileUpload component renders', () => {
    const props = {
      items: [],
      upload: () => {},
      remove: () => {}
    };

    const tree = renderer.create(
      <FileUpload {...props} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
