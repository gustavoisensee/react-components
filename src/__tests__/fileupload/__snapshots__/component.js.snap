exports[`FileUpload component test FileUpload component renders 1`] = `
<div
  className="content">
  <div
    className="content__input btn">
    <input
      className="uploader"
      id="18620171455559"
      multiple={true}
      onChange={[Function]}
      type="file" />
    <label
      className="uploader__title"
      htmlFor="18620171455559">
      Select file
    </label>
  </div>
  <div
    className="content__items" />
</div>
`;
